src/components/dashboard/Reason.js

```diff
@@ -15,12 +28,20 @@ export type AmountProps = {
   styles: any,
 }
 
+const PAYMENT_CATEGORY_1 = 'Digital Services'
```

Comment:
please use some enum-like solution to keep categories.
comparing some int codes and using them as some identifiers are better that strings

something like

```javascript
class PaymentCategory {
  static DigitalServices = 1
  static SocialMedia = 2
  static Product = 3
  static Course = 4
  static Donation = 5
  static Other = 6

  static labelOf(category) {
    const { DigitalServices, SocialMedia, Product, Course, Donation, Other } = this

     switch (category) {
      case DigitalServices:
        return 'DigitalServices'
      case SocialMedia:
        return 'Social Media Management'
      case Product:
        return 'Product'
      case Course:
        return 'Course / Private Consultation'
      case Donation:
        return 'Donation'
      case Other:
        return 'Other'
     }
  }
}
```

Reply:
what is the benefit of using 27 lines of code instead of using 6 lines of code to do the same thing?

Reply:
1. Readability 
2. Exact and strong definition. Enum items are grouped with some entity they defined for. In js it's very important because there's no types so any constant could be used everywhere.
3. It's a ready to use solution. Enumerated lists is a generic pattern existing in the all strong typed languages, first of all in C++:

```c
#define PAYMENT_CATEGORY_1 0 // bad design, just inserts 0 numeric literal so could be used everywhere we defined int argument

enum class PaymentCategory {
  SocialMedia,
  Donation,
  // ...
```

Yes JS isn't strong typed language but that doesn't means we must ignore all patterns. On the contrary we must use them to recoup this JS weakness

Reply:
the performance argument is a strong argument